imap_processing/cdf/utils.py

```diff
@@ -128,9 +128,17 @@ def write_cdf(
     dataset.attrs["ground_software_version"] = imap_processing._version.__version__
     # Add any parent files to the dataset attributes
     if parent_files:
-        # Include the current files if there are any and include just the filename
-        # [file1.txt, file2.cdf, ...]
-        dataset.attrs["Parents"] = dataset.attrs.get("Parents", []) + [
+        # Get the current value of "Parents" and ensure it is a list
```

Comment:
I am not sure why I got this error. I looked for any call to `write_cdf()` that was passing str but found none. I can look into this further.
```
Traceback (most recent call last):
  File "/Users/tech3371/Library/Caches/pypoetry/virtualenvs/imap-processing-52JUrpuW-py3.10/bin/imap_cli", line 6, in <module>
    sys.exit(main())
  File "/Users/tech3371/git/imap_processing/imap_processing/cli.py", line 872, in main
    instrument.process()
  File "/Users/tech3371/git/imap_processing/imap_processing/cli.py", line 345, in process
    self.post_processing(products)
  File "/Users/tech3371/git/imap_processing/imap_processing/cli.py", line 401, in post_processing
    products = [
  File "/Users/tech3371/git/imap_processing/imap_processing/cli.py", line 402, in <listcomp>
    write_cdf(dataset, parent_files=self._dependency_list)
  File "/Users/tech3371/git/imap_processing/imap_processing/cdf/utils.py", line 133, in write_cdf
    dataset.attrs["Parents"] = dataset.attrs.get("Parents", []) + [
TypeError: can only concatenate str (not "list") to str
```